### PR TITLE
Product listing

### PR DESCRIPTION
**Pull request by iprankurpandey**
Doen with product listing

### CODE REVIEW

File: Pages/ProductL/product.css [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+    background-color: silver;
+    max-width: 100%;
+    padding: 1rem;
+    position: fixed;
+    left: 0;
+    bottom: 0;
+    width: 100%
+   
+
+
```
Comment (gauravsinhaweb) [inline Pages/ProductL/product.css:30]: remove extra spacing in css

File: Pages/ProductL/product.css [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+.catagory-items-page {
+    display: grid;
+    grid-template-columns: 250px auto;
+    height: 100%;
+ 
+}
+
+.side-item-container {
+    /* margin: 1rem; */
```
[the remark this answers is not in the document]
Comment (gauravsinhaweb) [reply Pages/ProductL/product.css:64]: good to merge after changes!

**Review by gauravsinhaweb — CHANGES_REQUESTED** [review]

**Review by shangkaul — APPROVED** [review]
Looks good to merge!

File: Pages/ProductL/product.css (hunk 1 above)
Comment (iprankurpandey) [reply Pages/ProductL/product.css:30]: sure brother , will do that now .

**Pull request merged**